pkg/updatechecker/updatechecker.go

```diff
@@ -774,3 +793,47 @@ func isUpdateDeployable(updateCursor string, updates []upstreamtypes.Update) (bo
 	}
 	return true, ""
 }
+
+func GetAvailableAirgapUpdates(app *apptypes.App, license *kotsv1beta1.License) ([]types.AvailableUpdate, error) {
+	updates := []types.AvailableUpdate{}
+	if err := filepath.Walk(AvailableUpdatesDir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if info.IsDir() {
+			return nil
+		}
+
+		airgapMetadata, err := kotsutil.FindAirgapMetaInBundle(path)
+		if err != nil {
+			return errors.Wrap(err, "failed to find airgap metadata")
+		}
+
+		if airgapMetadata.Spec.AppSlug != license.Spec.AppSlug && airgapMetadata.Spec.ChannelID != license.Spec.ChannelID {
+			return nil
+		}
+
+		missingPrereqs, err := airgap.GetMissingRequiredVersions(app, airgapMetadata)
```

Comment:
this should just use the same `isUpdateDeployable` function above. `GetMissingRequiredVersions` is not applicable here as we don't check against deployment history anymore.